cmd/edit/addon/cmd.go

```diff
@@ -0,0 +1,226 @@
+		Build()
+	if err != nil {
+		reporter.Errorf("Failed to create AWS client: %v", err)
+		os.Exit(1)
+	}
+
+	awsCreator, err := awsClient.GetCreator()
+	if err != nil {
+		reporter.Errorf("Failed to get AWS creator: %v", err)
+		os.Exit(1)
+	}
+
+	// Create the client for the OCM API:
+	ocmConnection, err := ocm.NewConnection().
+		Logger(logger).
+		Build()
+	if err != nil {
+		reporter.Errorf("Failed to create OCM connection: %v", err)
+		os.Exit(1)
+	}
+	defer func() {
+		err = ocmConnection.Close()
+		if err != nil {
+			reporter.Errorf("Failed to close OCM connection: %v", err)
+		}
+	}()
+
+	// Get the client for the OCM collection of clusters:
+	ocmClient := ocmConnection.ClustersMgmt().V1()
+
+	// Try to find the cluster:
+	reporter.Debugf("Loading cluster '%s'", clusterKey)
+	cluster, err := clusterprovider.GetCluster(ocmClient.Clusters(), clusterKey, awsCreator.ARN)
+	if err != nil {
+		reporter.Errorf("Failed to get cluster '%s': %v", clusterKey, err)
+		os.Exit(1)
+	}
+
+	if cluster.State() != cmv1.ClusterStateReady {
+		reporter.Errorf("Cluster '%s' is not yet ready", clusterKey)
+		os.Exit(1)
+	}
+
+	parameters, err := clusterprovider.GetAddOnParameters(ocmClient.Addons(), addOnID)
+	if err != nil {
+		reporter.Errorf("Failed to get add-on '%s' parameters: %v", addOnID, err)
+		os.Exit(1)
+	}
+
+	addOnInstallation, err := clusterprovider.GetAddOnInstallation(ocmClient.Clusters(),
+		clusterKey, awsCreator.ARN, addOnID)
+	if err != nil {
+		reporter.Errorf("Failed to get add-on '%s' installation: %v", addOnID, err)
+		os.Exit(1)
+	}
+
+	var params []clusterprovider.AddOnParam
+	if parameters.Len() > 0 {
+		parameters.Each(func(param *cmv1.AddOnParameter) bool {
+			// Find the installation parameter corresponding to the addon parameter
+			var addOnInstallationParam *cmv1.AddOnInstallationParameter
+			addOnInstallation.Parameters().Each(func(p *cmv1.AddOnInstallationParameter) bool {
+				if p.ID() == param.ID() {
+					addOnInstallationParam = p
+					return false
+				}
+				return true
+			})
+			if !param.Editable() {
+				return true
+			}
+			input := interactive.Input{
+				Question: param.Name(),
+				Help:     param.Description(),
+				Required: param.Required(),
+			}
+
+			var val string
+			switch param.ValueType() {
+			case "boolean":
+				var boolVal bool
+				input.Default, _ = strconv.ParseBool(addOnInstallationParam.Value())
+				boolVal, err = interactive.GetBool(input)
+				if boolVal {
+					val = "true"
+				} else {
+					val = "false"
+				}
+			case "cidr":
+				var cidrVal net.IPNet
+				if addOnInstallationParam.Value() != "" {
+					_, defaultIDR, _ := net.ParseCIDR(addOnInstallationParam.Value())
+					input.Default = *defaultIDR
+				}
+				cidrVal, err = interactive.GetIPNet(input)
+				val = cidrVal.String()
+			case "number":
+				var numVal int
+				input.Default, _ = strconv.ParseInt(addOnInstallationParam.Value(), 10, 0)
+				numVal, err = interactive.GetInt(input)
+				val = strconv.Itoa(numVal)
```

Comment:
Backend would accept a float value as input, not sure if that matters.

Reply:
Thanks, I've added a new commit to convert numeric inputs to float